app/controllers/user.js

```diff
@@ -0,0 +1,19 @@
+const userService = require('../services/user');
+const logger = require('../logger');
+const errors = require('../errors');
+
+exports.createUser = async (req, res, next) => {
+  try {
+    const { body } = req;
+    const user = await userService.createUser(body);
+    if (user) {
+      logger.info(`User ${user.firstName} was created successfull`);
+      return res.status(200).send({ user });
```

Comment:
Remember that for good practices when we return a single user we do not add the key that represents the entity, we only return its properties, and be careful that this way you are also returning the password.